doc/ref_model/chapters/chapter02.md

```diff
@@ -170,8 +178,9 @@ By trying to sort workloads into different categories based on the requirements
 # 2.3 Cloud Infrastructure Profiles
 
 Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
-- **Basic**: for Workloads that can tolerate resource over-subscription and variable latency.
-- **Network Intensive**: for Workloads that require predictable computing performance, high network throughput and low network latency.
+- **Basic**: For Workloads that can tolerate resource over-subscription and variable latency.
+- **Network Intensive**: For Workloads that require predictable computing performance, high network throughput and low network latency.
+- **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions , very low network latency ,very high storage requirements and very high computing requirements "GPU" .
```

Comment:
Do we really need GPU-s on the edge? For me GPU-s are more for ML training and I do not see an use case for this on the edge.

Reply:
@CsatariGergely Hello Gergely, we have the use case of the remote Edge Survellience with video analytics. Where the live streams need to be analyzed at the Edge (ML-assisted) and alarms are raised for the suspected scenes. It is one of the 5G use cases for Mission Critical IoT.